Makefile

```diff
@@ -151,31 +158,34 @@ install-deps-Linux: ## Install deps on GNU/Linux
 install-deps-Darwin: ## Install deps on Mac OSX
 	brew install $(PKG_LIB_TEST)
 
-install-libraries-deps: ## Install deps on GNU/Linux
-	if [[ "$(UNAME_S)" == "Linux" ]]; then (cd build && wget --no-check-certificate https://cmake.org/files/v3.3/cmake-3.3.2-Linux-x86_64.tar.gz && echo "f3546812c11ce7f5d64dc132a566b749 *cmake-3.3.2-Linux-x86_64.tar.gz" > cmake_md5.txt && md5sum -c cmake_md5.txt && tar -xvf cmake-3.3.2-Linux-x86_64.tar.gz > /dev/null && mv cmake-3.3.2-Linux-x86_64 cmake-install && PATH=$(pwd)/build/cmake-install:$(pwd)/build/cmake-install/bin:$PATH ) ; fi
-	(cd build && wget http://curl.haxx.se/download/curl-7.58.0.tar.gz && tar -xvf curl-7.58.0.tar.gz && cd curl-7.58.0/ && bash ./configure && make && sudo make install)
-	if [[ "$(UNAME_S)" == "Darwin" ]]; then brew install curl ; fi
-	# install uncrustify
-	(cd build && git clone https://github.com/uncrustify/uncrustify.git)
-	(cd build/uncrustify && mkdir build && cd build && cmake .. && make && sudo make install)
-
 install-linters: install-linters-$(UNAME_S) ## Install linters
 	go get -u github.com/FiloSottile/vendorcheck
 	# For some reason this install method is not recommended, see https://github.com/golangci/golangci-lint#install
 	# However, they suggest `curl ... | bash` which we should not do
 	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
 	VERSION=1.10.2 ./ci-scripts/install-golangci-lint.sh
 
-install-deps-libc: install-deps-libc-$(OSNAME) install-libraries-deps
+install-deps-skyapi-linux:
+	sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
+	sudo apt-get update
+	sudo apt-get install cmake
+	sudo apt-get install libcurl3-gnutls
+
+install-deps-skyapi-osx:
+	brew list cmake || brew install cmake
+	brew list curl || brew install curl
+
+install-deps-libc: install-deps-libc-$(OSNAME)
```

Comment:
Instead of `OSNAME` use `UNAME_S` for target names .

Reply:
Done in last commits